### PR TITLE
servertype for client 2017 + 

### PR DESCRIPTION
**Pull request by sctnightcore**
client : http://herc.ws/board/topic/15348-kro-unpacked-clients-releases/#comment-85073
packet : https://gitlab.com/4144/servergreps/
@moneyisgod 

```
fix send RagexeRE_2017_06_14b : ok
RagexeRE_2017_06_07c.pm : ok 
RagexeRE_2017_09_13b.pm : ok
RagexeRE_2017_09_20b.pm : ok
new charblocksize : 147 it WORK ! 
```
**New REV !** 
```
'0ACC' => ['exp', 'a4 V v2', [qw(ID val type flag)]],
'0ACB' => ['stat_info', 'v V', [qw(type val)]],
```
**My problem**
```
1.cannot get packet from peek ! 
2.starting from 2017-08   new charblocksize !  
https://github.com/rathena/rathena/pull/2458/files
```
image

### CODE REVIEW

**Comment by sctnightcore** [comment]
@lututui I think so

**Comment by sctnightcore** [comment]
Waiting for hercules and rathena to update server. To test servertype !

File: src/Network/Send/kRO/RagexeRE_2017_09_13b.pm [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		'08AC' => ['actor_look_at', 'v C', [qw(head body)]],
+		'095C' => ['actor_name_request', 'a4', [qw(ID)]],
+		'0927' => ['buy_bulk_buyer', 'a4 a4 a*', [qw(buyerID buyingStoreID itemInfo)]], #Buying store
+		'085A' => ['buy_bulk_closeShop'],			
+		'0866' => ['buy_bulk_openShop', 'a4 c a*', [qw(limitZeny result itemInfo)]], #Selling store
+		'08A6' => ['buy_bulk_request', 'a4', [qw(ID)]], #6
+		'0923' => ['character_move', 'a3', [qw(coordString)]],
+		'035F' => ['friend_request', 'a*', [qw(username)]],# len 26
+		'088C' => ['homunculus_command', 'v C', [qw(commandType, commandID)]],
+		'091D' => ['item_drop', 'v2', [qw(index amount)]],
```
Comment (moneyisgod) [inline src/Network/Send/kRO/RagexeRE_2017_09_13b.pm]: ['item_drop', 'a2 v', [qw(ID amount)]],#6

File: src/Network/Send/kRO/RagexeRE_2017_06_14b.pm [hunk 2]
```diff
@@ -9,44 +9,78 @@
...
+		'087E' => ['actor_look_at', 'v C', [qw(head body)]],
+		'087D' => ['actor_name_request', 'a4', [qw(ID)]],
+		'092F' => ['buy_bulk_buyer', 'a4 a4 a*', [qw(buyerID buyingStoreID itemInfo)]], #Buying store
+		'086B' => ['buy_bulk_closeShop'],			
+		'08A2' => ['buy_bulk_openShop', 'a4 c a*', [qw(limitZeny result itemInfo)]], #Selling store
+		'0860' => ['buy_bulk_request', 'a4', [qw(ID)]], #6
+		'0361' => ['character_move', 'a3', [qw(coordString)]],
+		'0867' => ['friend_request', 'a*', [qw(username)]],# len 26
+		'0364' => ['homunculus_command', 'v C', [qw(commandType, commandID)]],
+		'0367' => ['item_drop', 'v2', [qw(index amount)]],
```
Comment (moneyisgod) [inline src/Network/Send/kRO/RagexeRE_2017_06_14b.pm]: ['item_drop', 'a2 v', [qw(ID amount)]],#6

File: src/Network/Send/kRO/RagexeRE_2017_06_07c.pm [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+		'085A' => ['actor_look_at', 'v C', [qw(head body)]],
+		'0944' => ['actor_name_request', 'a4', [qw(ID)]],
+		'0919' => ['buy_bulk_buyer', 'a4 a4 a*', [qw(buyerID buyingStoreID itemInfo)]], #Buying store
+		'093D' => ['buy_bulk_closeShop'],			
+		'0949' => ['buy_bulk_openShop', 'a4 c a*', [qw(limitZeny result itemInfo)]], #Selling store
+		'0863' => ['buy_bulk_request', 'a4', [qw(ID)]], #6
+		'0934' => ['character_move', 'a3', [qw(coordString)]],
+		'0885' => ['friend_request', 'a*', [qw(username)]],# len 26
+		'0942' => ['homunculus_command', 'v C', [qw(commandType, commandID)]],
+		'0864' => ['item_drop', 'v2', [qw(index amount)]],
```
Comment (moneyisgod) [inline src/Network/Send/kRO/RagexeRE_2017_06_07c.pm]: ['item_drop', 'a2 v', [qw(ID amount)]],#6

**Comment by sctnightcore** [comment]
@moneyisgod  omg ! WTF ty 
I am fixing it

**Comment by moneyisgod** [comment]
RagexeRE_2016_01_06a.pm
RagexeRE_2016_02_03a.pm
RagexeRE_2016_12_28a.pm
RagexeRE_2017_01_25a.pm
RagexeRE_2017_05_17a.pm
RagexeRE_2017_06_14b.pm

All contains wrong syntax for item_drop.

**Comment by sctnightcore** [comment]
starting from 2017-08   new charblocksize !   @lututui  @moneyisgod  @alisonrag  @Henrybk 
https://github.com/rathena/rathena/pull/2458/files 
Who can help me?
I do not have any knowledge about charblocksize

**Comment by godwins530** [comment]
@sctnightcore On client 20170614,can not see party members,could you fix it ?thanks

**Pull request merged**